packages/flash-swap/test/shared/fixtures.ts

```diff
@@ -23,10 +23,10 @@ import { deployGodModeErc20 } from "./deployers";
 type IntegrationFixtureReturnType = {
   balanceSheet: BalanceSheetV1;
   fintroller: FintrollerV1;
-  hifiFlashUniswapV2: HifiFlashUniswapV2;
-  hifiFlashUniswapV2Underlying: HifiFlashUniswapV2Underlying;
+  collateralFlashUniswapV2: CollateralFlashUniswapV2;
```

Comment:
@paulrberg would be great to re-order alphabetically and place collateralFlashUniswapV2 right below the balanceSheet variable.